CHANGELOG.MD

```diff
@@ -0,0 +1,5 @@
+# Changelog
+
+## **1.5.0**
+- Migration from jCenter to Maven Central 🎉
+- ‼️ Important ‼️ groupId has changed. New groupId is `io.github.ackeecz`.
```

Comment:
Isn't artifactId changed as well?

Reply:
no, I changed that to `ackee-gradle-plugin` initially but then I realised that I actually don' know why it is named like it is and that it might have a good reason + less breaking changes